modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -148,10 +167,22 @@ private function sortSeries(&$series, $sort_order)
         return ($a['episode_id'] < $b['episode_id']) ? -1 : 1;
       }
 
-      return ($b['episode_id'] > $a['episode_id']) ? 1 : -1;
+      return ($a['episode_id'] < $b['episode_id']) ? 1 : -1;
     });
 
-    return $series;
+    usort($no_episode, function ($a, $b) use ($sort_order) {
+      if ($a['episode_id'] == $b['episode_id']) {
```

Comment:
Do we need this? given this sort function uses content without an episode number

Reply:
It should be using `date` as the comparison and not `episode_id`